astropy/units/physical.py

```diff
@@ -210,6 +210,9 @@ class PhysicalType:
     `~astropy.units.core.UnitBase.physical_type` attribute of units.
     This class is not intended to be instantiated directly in user code.
 
+    To learn what physical types are available, please refer to the
```

Comment:
I'd keep this shorter perhaps "For a list of physical types, see `astropy.units.physical`." (and same in other instances)

Reply:
Good idea!  Let's go with your wording.